src/com/lilithsthrone/game/character/fetishes/Fetish.java

```diff
@@ -30,10 +30,6 @@
  */
 public enum Fetish {
 	
-	// FETISHES:
-
-	// Sex types:
-	
```

Comment:
Is this removed intentionally?

Reply:
I don't remember removing this but it's 3am
Looks like legacy comments from when Fetish.java was created dating back to 2017